architecture/infrastructure_components/image_registry.adoc

```diff
@@ -10,16 +10,17 @@
 toc::[]
 
 == Overview
-OpenShift utilizes any server implementing the Docker registry API as a source
-of images, including the canonical Docker Hub, private registries run by third
-parties, and the integrated OpenShift registry.
+OpenShift can utilize any server implementing the Docker registry
+API as a source of images, including the canonical Docker Hub, private
+registries run by third parties, and the integrated OpenShift registry.
 
 [[integrated-openshift-registry]]
 
 == Integrated OpenShift Registry
 OpenShift provides an integrated Docker registry that adds the ability to
 provision new image repositories on the fly. This allows users to automatically
-have a place for their builds to push the resulting images.
+have a place for their
+link:../core_objects/builds.html[builds] to push the resulting images.
```

Comment:
Just noticed this is a ref to the old structure. Pull your PR into https://github.com/openshift/openshift-docs/pull/739 to fix and merge.